File: Eldar Dautović/ridewithme/UI/ridewithme_mobile/lib/models/faq.dart
// -- faq.dart --
import 'package:json_annotation/json_annotation.dart';
import 'package:ridewithme_mobile/models/korisnik.dart';

part 'faq.g.dart';

@JsonSerializable()
class FAQ {
  int? id;
  String? pitanje;
  String? odgovor;
  DateTime? datumKreiranja;
  DateTime? datumIzmjene;
  Korisnik? korisnik;

  FAQ(
    this.id,
    this.pitanje,
    this.odgovor,
    this.datumKreiranja,
    this.datumIzmjene,
    this.korisnik,
  );

  factory FAQ.fromJson(Map<String, dynamic> json) => _$FAQFromJson(json);

  Map<String, dynamic> toJson() => _$FAQToJson(this);
}

File: Eldar Dautović/ridewithme/ridewithme.Model/Models/FAQ.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ridewithme.Model.Models
{
    public class FAQ
    {
        public int Id { get; set; }

        public string Pitanje { get; set; }

        public string Odgovor { get; set; }

        public DateTime DatumKreiranja { get; set; }

        public DateTime DatumIzmjene { get; set; }

        public Korisnici Korisnik { get; set; }
    }
}
